scene/3d/vehicle_body_3d.h

```diff
@@ -42,51 +42,51 @@ class VehicleWheel3D : public Node3D {
 
 	Transform m_worldTransform;
 	Transform local_xform;
-	bool engine_traction;
-	bool steers;
+	bool engine_traction = false;
+	bool steers = false;
 
 	Vector3 m_chassisConnectionPointCS; //const
 	Vector3 m_wheelDirectionCS; //const
 	Vector3 m_wheelAxleCS; // const or modified by steering
 
-	real_t m_suspensionRestLength;
-	real_t m_maxSuspensionTravelCm;
-	real_t m_wheelRadius;
+	real_t m_suspensionRestLength = 0.15;
+	real_t m_maxSuspensionTravelCm = 500;
+	real_t m_wheelRadius = 0.5;
 
-	real_t m_suspensionStiffness;
-	real_t m_wheelsDampingCompression;
-	real_t m_wheelsDampingRelaxation;
-	real_t m_frictionSlip;
-	real_t m_maxSuspensionForce;
-	bool m_bIsFrontWheel;
+	real_t m_suspensionStiffness = 5.88;
+	real_t m_wheelsDampingCompression = 0.83;
+	real_t m_wheelsDampingRelaxation = 0.88;
+	real_t m_frictionSlip = 10.5;
+	real_t m_maxSuspensionForce = 6000;
+	bool m_bIsFrontWheel = false;
 
-	VehicleBody3D *body;
+	VehicleBody3D *body = nullptr;
 
 	//btVector3	m_wheelAxleCS; // const or modified by steering ?
 
-	real_t m_steering;
-	real_t m_rotation;
-	real_t m_deltaRotation;
-	real_t m_rpm;
-	real_t m_rollInfluence;
-	real_t m_engineForce;
-	real_t m_brake;
+	real_t m_steering = real_t(0.);
+	real_t m_rotation = real_t(0.);
+	real_t m_deltaRotation = real_t(0.);
+	real_t m_rpm = 0;
+	real_t m_rollInfluence = real_t(0.);
+	real_t m_engineForce = real_t(0.);
+	real_t m_brake = real_t(0.);
```

Comment:
I know it was in the constructor but I don't see why the explicit `real_t()` cast is needed.